internal/inspect/inspect.go

```diff
@@ -1,104 +1,123 @@
 package inspect
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
+	"os"
 	"sort"
 	"strings"
 	"text/tabwriter"
 
 	"github.com/deislabs/cnab-go/bundle"
+	"github.com/docker/app/internal"
 	"github.com/docker/app/render"
 	"github.com/docker/app/types"
+	"github.com/docker/app/types/metadata"
 	"github.com/docker/app/types/parameters"
 	composetypes "github.com/docker/cli/cli/compose/types"
 	units "github.com/docker/go-units"
 )
 
+type service struct {
+	Name     string
```

Comment:
I think you should rename the parameters for the json marshaling, with lowercase
```go
type service struct {
    Name string `json:"name,omitempty`
...
}
```

Reply:
The main `docker inspect` command doesn't use the "standard" way.